src/human-readable/types/signatures.ts

```diff
@@ -207,6 +213,12 @@ export type SolidityKeywords =
   | 'var'
   | 'view'
   | 'virtual'
+  | `address${`[${string}]` | ''}`
```

Comment:
Workaround for the previous `Abitype` union

Reply:
works for me!